charts/argocd-config/templates/app-of-apps.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: app-of-apps
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: argocd
+  project: {{ .Values.environment }}
+  source:
+    path: charts/argocd-apps
+    repoURL: {{ .Values.repoUrls.deploy }}
+    targetRevision: de/argo-ui-simple # debug! needs to be set to HEAD
```

this needs to change before merging
```suggestion
    targetRevision: HEAD
```